client/CMessage.cpp

```diff
@@ -296,6 +296,8 @@ void CMessage::drawIWindow(CInfoWindow * ret, std::string text, PlayerColor play
 
 void CMessage::drawBorder(PlayerColor playerColor, SDL_Surface * ret, int w, int h, int x, int y)
 {
+	if(playerColor == PlayerColor::SPECTATOR)
+		playerColor = PlayerColor(1);

```

Comment:
This chunk is duplicated many times. Consider adding a function|method like
 `PlayerColor playerColorToColor(const PlayerColor & color)` 
somewhere